### PR TITLE
Update comments for chip tests

### PR DESCRIPTION
**Pull request by royarg02**
This PR updates some comments of tests for the `Chip` widgets.

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: packages/flutter/test/material/chip_test.dart [hunk 1]
```diff
@@ -3497,28 +3497,28 @@ void main() {
...
         useDeleteButtonTooltip: false,
       ),
     );
 
-    // Tap at the delete icon of the chip, which is at the right
-    // side of the chip
-    final Offset topRightOfInkwell = tester.getTopLeft(find.byType(InkWell).first);
-    final Offset tapLocationOfDeleteButton = topRightOfInkwell + const Offset(8, 8);
-    final TestGesture tapGesture = await tester.startGesture(tapLocationOfDeleteButton);
+    // Hover over the delete icon of the chip
```
Comment (goderbauer) [inline packages/flutter/test/material/chip_test.dart:3509]: Why do you want to change this test?

Comment (RoyARG02) [reply packages/flutter/test/material/chip_test.dart:3509]: The following tests follow almost similar procedure to test the same functionality, so this change is to make all tests have similar code.

Comment (goderbauer) [reply packages/flutter/test/material/chip_test.dart:3509]: This is reducing our test coverage, though. The chip needs to support both modes of interaction: pressing and holding as well as hovering - and we should check for both.

Comment (RoyARG02) [reply packages/flutter/test/material/chip_test.dart:3509]: Okay, I'll revert the change.

**Review by goderbauer — APPROVED** [review]
LGTM

**Review by HansMuller — APPROVED** [review]
LGTM

**Pull request merged**